src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -477,7 +478,7 @@ TEST_P(conv_fp32_scale, basic) {
         reorder("reorder_bfyx", "scale", p.default_format, data_types::f32)
     );
 
-    tolerance = 1e-5f;
+    tolerance = p.default_type==data_types::f16 ? 1e-3f : 1e-5f;
```

Comment:
I think it may make sense to add common function for tolerance value, something like `get_default_tolerance(cldnn::data_types)`

Reply:
Thank you for your great suggestion. Done.